tributech-solutions/tributech-dsk-docs
# Feature/improve iot provide doc (DSK-1840, DSK-1841)

feat: MQTT source doc (DSK-1840) 

* better sample values at MQTT value sources
* test and verify at pre-defined MQTT topics
* fixed id generator fiddle

---

feat: opcua source doc (DSK-1841) 

* use PublishingInterval 1000 as sample (experience from Findustrial)
* add verify step (also for simulated sensor)
